build.gradle

```diff
@@ -19,8 +19,8 @@ buildscript {
 
 allprojects {
     repositories {
-        jcenter()
         google()
+        jcenter()
```

Comment:
@ethan627hsu Why are you swapping these lines? This issue can be resolved by adding `multi-dex` dependency.

Reply:
The swapping prevents an error when referencing recyclerview in gradle

Reply:
@ethan627hsu Can you add the reference/link from where you got this solution?